treeherder/model/derived/jobs.py

```diff
@@ -1901,7 +1901,7 @@ def schedule_log_parsing(self, log_data, priority="normal"):
             tasks[log_obj["job_guid"]].append({
                 "func_name": func_name,
                 "routing_key": routing_key,
-                "job_log_url": log_obj
+                "job_log_url": log_obj['url']
```

Comment:
Do we need to make this change in a deploy prior to removing support for handling the object in `expand_log_url()`?